tests/e2e_test.go

```diff
@@ -389,7 +389,7 @@ func TestPublishSuccess(t *testing.T) {
 	if err := app.SnapshotCmd(ctx, td); err != nil {
 		t.Fatalf("expected Snapshot command to pass, got err: %s", err)
 	}
-	snapshotSigner, err := keys.GetSigningKey(ctx, snapshotKey)
+	snapshotSigner, err := keys.GetSigningKey(ctx, snapshotKey, true)
```

Comment:
any test where deprecated is false, or is that coming later?

Reply:
Not e2e yet, just in the package tests. In e2e we only expect deprecated keys (until Part 3, coming soon) because that's what the CLI hard-codes


Part 3 will have some refactor for CLI to parametrize/change the encoding/decoding